Source/Testing/Aggregates/AggregateRootAssertion.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Dolittle.SDK.Aggregates;
+using System;
+using System.Linq;
+using Dolittle.SDK.Testing.Aggregates.Events;
+
+namespace Dolittle.SDK.Testing.Aggregates;
+
+/// <summary>
+/// Represents the assertions that can be performed on an <see cref="AggregateRoot"/>.
+/// </summary>
+public class AggregateRootAssertion
```

Comment:
This only asserts on type and count of the events. How do we verify that the details of an event is correct?